run-tests.php

```diff
@@ -2476,6 +2476,9 @@ function run_test($php, $file, $env)
                 $clean_params = array();
                 settings2array($ini_overwrites, $clean_params);
                 $clean_params = settings2params($clean_params);
+                if (substr(PHP_OS, 0, 3) === "WIN" && $workerID) {
```

Comment:
I used `substr(PHP_OS, 0, 3) === "WIN"` like below (instead of `PHP_OS_FAMILY`); in master we can use `IS_WINDOWS` instead.

Reply:
What do you think about using `$orig_ini_settings` instead of `$clean_params` here? Those are the ini settings that get passed to SKIPIF as well (basically everything apart from `--INI--`).

Reply:
That sounds like a very good idea! :)